test/db-migrations/20250910-02-backfill-last-login-at.spec.js

```diff
@@ -32,19 +32,40 @@ describeMigration('20250910-02-backfill-last-login-at', ({ runMigrationBeingTest
       { actorId: 2, action: 'user.session.create', acteeId: 'actee2', loggedAt: new Date('2025-01-08T16:45:00Z') },
     );
 
+    // Delete Alice
+    await db.any(sql`UPDATE actors SET "deletedAt" = '2025-01-01' WHERE id = 1`);
```

Comment:
Since #1186, I think we've been trying to add times and time zones to timestamps.

```suggestion
    await db.any(sql`UPDATE actors SET "deletedAt" = '2025-01-01T00:00:00Z' WHERE id = 1`);
```